extensions/incubator/src/main/java/io/opentelemetry/extension/incubator/trace/Propagation.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.extension.incubator.trace;
```

Comment:
There's a `io.opentelemetry.extension.propagation` package which would probably be more appropriate.

Reply:
moved to `io.opentelemetry.extension.incubator.propagation`